config/initializers/archives_space_searcher_config.rb

```diff
@@ -0,0 +1,11 @@
+# Try to load a local version of the config file if it exists -
+# expected to be in quicksearch
+# _root/config/searchers/<my_searcher_name>_config.yml
+
+config_file = [
+  File.join(Rails.root, '/config/searchers/archives_space_config.yml'),
+  File.expand_path('../archives_space_config.yml', __dir__)
+].first { |file| File.exists? file }
+
+QuickSearch::Engine::ARCHIVESSPACE_CONFIG =
```

Comment:
For naming consistency, please rename "ARCHIVESSPACE_CONFIG" to "ARCHIVES_SPACE_CONFIG" in this file and in app/searchers/quick_search/archives_space_searcher.rb